torch/distributed/_sharded_tensor/__init__.py

```diff
@@ -289,11 +289,8 @@ def init_from_local_shards(
     Args:
         local_shards (List[:class `torch.distributed._sharded_tensor.Shard`]): A list
             of shards that represent the local shards on this rank.
-        sharded_tensor_metadata (:class:`torch.distributed._sharded_tensor.ShardedTensorMetadata`)
-            The ShardedTensorMetadata that created manually, represents the global metadata
-            of the ShardedTensor, must comply with `local_shards` defined in each rank.
-            Note that `sharded_tensor_metadata` must be valid and should also contain
-            local shards metadata.
+        overall_size (int...):  a list, tuple, or `torch.Size` of integers defining the
+            shape of the overall sharded tensor.
```

Comment:
probably add an example to showcase how to use it?

Example:

rank-0
- local_shards  =...
- global_size = (10, 20)
- sharded_tensor = ShardedTensor.init_from_local(local_shards)

rank-1
- local_shards  =...
- global_size = (10, 20)
- sharded_tensor = ShardedTensor.init_from_local(local_shards)